plugins/mcreator-core/datalists/blocksitems.yaml

```diff
@@ -4074,6 +4137,407 @@
+  type: block
+
+- Blocks.LIGHTNING_ROD:
+  readable_name: "Lightning Rod"
+  type: block
+
+- Blocks.POINTED_DRIPSTONE:
+  readable_name: "Pointed Dripstone"
+  type: block
+
+- Blocks.DRIPSTONE_BLOCK:
+  readable_name: "Dripstone Block"
+  type: block
+
+- Blocks.CAVE_VINES:
+  readable_name: "Cave Vines"
+  type: block
+
+- Blocks.CAVE_VINES:
+  readable_name: "Cave Vines Plant"
+  type: block
+
+- Blocks.SPORE_BLOSSOM:
+  readable_name: "Spore Blossom"
+  type: block
+
+- Blocks.AZALEA:
+  readable_name: "Azalea"
+  type: block
+
+- Blocks.FLOWERING_AZALEA:
+  readable_name: "Flowering Azalea"
+  type: block
+
+- Blocks.MOSS_CARPET:
+  readable_name: "Moss Carpet"
+  type: block
+
+- Blocks.MOSS_BLOCK:
+  readable_name: "Moss Block"
+  type: block
+
+- Blocks.BIG_DRIPLEAF:
+  readable_name: "Big Dripleaf"
+  type: block
+
+- Blocks.BIG_DRIPLEAF_STEM:
+  readable_name: "Big Dripleaf Stem"
+  type: block
+
+- Blocks.SMALL_DRIPLEAF:
+  readable_name: "Small Dripleaf"
+  type: block
+
+- Blocks.HANGING_ROOTS:
+  readable_name: "Hanging Roots"
+  type: block
+
+- Blocks.ROOTED_DIRT:
+  readable_name: "Rooted Dirt"
+  type: block
+
+- Blocks.DEEPSLATE:
+  readable_name: "Deepslate"
+  type: block
+
+- Blocks.COBBLED_DEEPSLATE:
+  readable_name: "Cobbled Deepslate"
+  type: block
+
+- Blocks.COBBLED_DEEPSLATE_STAIRS:
+  readable_name: "Cobbled Deepslate Stairs"
+  type: block
+
+- Blocks.COBBLED_DEEPSLATE_SLAB:
+  readable_name: "Cobbled Deepslate Slab"
+  type: block
+
+- Blocks.COBBLED_DEEPSLATE_WALL:
+  readable_name: "Cobbled Deepslate Wall"
+  type: block
+
+- Blocks.POLISHED_DEEPSLATE:
+  readable_name: "Polished Deepslate"
+  type: block
+
+- Blocks.POLISHED_DEEPSLATE_STAIRS:
+  readable_name: "Polished Deepslate Stairs"
+  type: block
+
+- Blocks.POLISHED_DEEPSLATE_SLAB:
+  readable_name: "Polished Deepslate Slab"
+  type: block
+
+- Blocks.POLISHED_DEEPSLATE_WALL:
+  readable_name: "Polished Deepslate Wall"
+  type: block
+
+- Blocks.DEEPSLATE_TILES:
+  readable_name: "Deepslate Tiles"
+  type: block
+
+- Blocks.DEEPSLATE_TILE_STAIRS:
+  readable_name: "Deepslate Tile Stairs"
+  type: block
+
+- Blocks.DEEPSLATE_TILE_SLAB:
+  readable_name: "Deepslate Tile Slab"
+  type: block
+
+- Blocks.DEEPSLATE_TILE_WALL:
+  readable_name: "Deepslate Tile Wall"
+  type: block
+
+- Blocks.DEEPSLATE_BRICKS:
+  readable_name: "Deepslate Bricks"
+  type: block
+
+- Blocks.DEEPSLATE_BRICKS_STAIRS:
+  readable_name: "Deepslate Bricks Stairs"
+  type: block
+
+- Blocks.DEEPSLATE_BRICKS_SLAB:
+  readable_name: "Deepslate Bricks Slab"
+  type: block
+
+- Blocks.DEEPSLATE_BRICKS_WALL:
+  readable_name: "Deepslate Bricks Wall"
+  type: block
+
+- Blocks.CHISELED_DEEPSLATE:
+  readable_name: "Chiseled Deepslate"
+  type: block
+
+- Blocks.CRACKED_DEEPSLATE_BRICKS:
+  readable_name: "Cracked Deepslate Bricks"
+  type: block
+
+- Blocks.CRACKED_DEEPSLATE_TILES:
+  readable_name: "Cracked Deepslate Tiles"
+  type: block
+
+- Blocks.INFESTED_DEEPSLATE:

```

Comment:
I'd rename this to MONSTER_EGG#6 for the sake of consistency and since they probably won't add many more.